ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Selection-Tool/bond-properties.spec.ts

```diff
@@ -0,0 +1,599 @@
+
+      for (let i = 0; i < rCOptions.length - 1; i++) {
+        await doubleClickOnBond(page, BondType.SINGLE, i);
+        await page.getByText('Unmarked').click();
+        await page
+          .getByRole('option', { name: rCOptions[i], exact: true })
+          .click();
+        await pressButton(page, 'Apply');
+      }
+
+      await doubleClickOnBond(page, BondType.SINGLE, 8);
+      await page.getByText('Unmarked').click();
+      await page.getByRole('option', { name: 'Center', exact: true }).click();
+      await pressButton(page, 'Apply');
+
+      await doubleClickOnBond(page, BondType.SINGLE, 8);
+      await page.getByText('Center', { exact: true }).click();
+      await page.getByRole('option', { name: 'Unmarked', exact: true }).click();
+      await pressButton(page, 'Apply');
+
+      const expectedFile = await getRxn(page, 'v2000');
+      await saveToFile('Rxn/rxn_1463_to_open-expected.rxn', expectedFile);
+
+      const METADATA_STRING_INDEX = [2, 7, 25, 40, 66];
+
+      const { fileExpected: rxnFileExpected, file: rxnFile } =
+        await receiveFileComparisonData({
+          page,
+          expectedFileName: 'tests/test-data/Rxn/rxn_1463_to_open-expected.rxn',
+          metaDataIndexes: METADATA_STRING_INDEX,
+          fileFormat: 'v2000',
+        });
+
+      expect(rxnFile).toEqual(rxnFileExpected);
+    },
+  );
+
+  test.fixme(
+    `Change 'Reacting Center' field value - 2/2 open and edit`,
+    async ({ page }) => {
+      /*
+        Test case: EPMLSOPKET-1463(2)
+        Description: Open and edit
+
+        Now it doesn`t work because if we try to save bond with 'Not center' option, 
+        we can`t  open the file
+    */
+
+      await openFileAndAddToCanvas('Rxn/rxn_1463_to_open-expected.rxn', page);
+      await doubleClickOnBond(page, BondType.SINGLE, 8);
+      await page.getByText('Unmarked').click();
+      await page.getByRole('option', { name: 'Center', exact: true }).click();
+      await pressButton(page, 'Apply');
+    },
+  );
+
+  test('Apply for all selected bonds', async ({ page }) => {
+    /*
+        Test case: EPMLSOPKET-2926
+        Description: All structure is highlighted green color.
+        'Bond properties' window is opened.  
+        All selected bonds are changed with 'Double' bond.
+        'Bond properties' window is opened. 
+        All selected bonds are changed. Bonds that were not selected are not changed.
+        Selected bonds are highlighted green color.
+        'Bond properties' window is opened. 
+        All selected bonds are changed. Bonds that were not selected are not changed.
+        Selected bonds are highlighted green color.
+ */
+
+    await openFileAndAddToCanvas('Molfiles-V2000/mol_2926_to_open.mol', page);
+    await selectAllStructuresOnCanvas(page);
+    await doubleClickOnBond(page, BondType.SINGLE, 1);
+    await page.getByText('Single').click();
+    await page.getByRole('option', { name: 'Double', exact: true }).click();
```

Comment:
I think we can move this part into separate function, because it repeates too many times.

Reply:
Good point, tnx